.drone.yml

```diff
@@ -4,7 +4,8 @@ workspace:
 
 services:
   web:
-    image: fpfis/httpd-php-dev:7.1
+    # @todo: move back to 7.1 after Docker image is fixed upstream. Issue related to missed apache mod_rewrite module.
```

Comment:
@sergepavle please add a reference to some ticket/issue where we can track when it's solved.

Reply:
This has been added and reverted later, so, discard this please.